### DISCUSSION TITLE
Virus detected!

### DISCUSSION THREAD

**Discussion by MvPGEO** [SnosMe/awakened-poe-trade#1296] [Q&A]
![image](https://github.com/SnosMe/awakened-poe-trade/assets/20114377/61656f27-615f-40a9-aa27-4b784ffb57f2)
this never happened before but with last update now i get this.

**Comment by SnosMe** [comment]
https://www.reddit.com/r/pathofexile/comments/1av1ed1/anyone_using_awakened_poe_trade_is_this_a_false/
